Neos.Neos/Classes/Fusion/ConvertUrisImplementation.php

```diff
@@ -114,16 +115,18 @@ public function evaluate()
             ), 1382624087);
         }
 
+        /** @var RenderingMode $renderingMode */
+        $renderingMode = $this->runtime->fusionGlobals->get('renderingMode');
```

Comment:
I probably mix things up, but didn't we add a method to the Fusion runtime to retrieve those things in a type-safe manner?

Reply:
nope this discussion lead into a rabbit hole and we decided to continue with more critical things

Reply:
OK, I still think that a separate method `FusionGlobals::getTyped(string $name, string $type)` would do no harm, but I didn't mean to re-open that discussion, sry

Reply:
no problem. The discussion is of course not closed indefinitely but to get the `FusionGlobal` change in at all we kept it simple.

The issue can be found here: https://github.com/neos/neos-development-collection/issues/4475

Its just not something i plan on working atm but i guess we can pick up the work with 9.1